03-task-manager/answers/db/connect.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require('mongoose');
+
+const connectDB = (url) => {
+	return mongoose
+		.connect(url, {
+			writeConcern: {
+				w: 'majority',
+			},
+		})
```

Comment:
Just curious, were these the options that were passed into the connect call in the video? Mainly asking because I've mainly been seeing the following in other assignments, so I'm curious why this one differs:

```
-    useNewUrlParser: true,
-    useCreateIndex: true,
-    useFindAndModify: false,
-    useUnifiedTopology: true
```

The `writeConcern` option you've included here is fine but generally not something I'd be too worried about including at this stage. It tells MongoDb how to handle requests to create or edit data in the database if you have multiple copies (replicas) of the database running

Reply:
Thanks for the review. I was getting an error (MongoWriteConcernError) - I saved the error message and looked it up and found that I could update my MongoDB connection configuration with this code  to prevent the error. Perhaps I had multiple copies of the database running?